scripts/AudioManager.js

```diff
@@ -12,6 +12,23 @@ function AudioManager(addEventListener, isTownTune) {
 	function playHourlyMusic(hour, game, isHourChange) {
 		audio.loop = true;
 		audio.removeEventListener("ended", playKKSong);
+		if (game == 'mix-all') {
```

Comment:
I feel like this switching should be handled in StateManager instead of AudioManager. In my eyes, AudioManager should be treated as a dumb view, and this gives it too much knowledge and responsibility. With this structure, while using mix-all, the view knows what song is playing, while the model does not. That doesn't feel right to me.